src/transformers/convert_slow_tokenizer.py

```diff
@@ -832,13 +833,34 @@ def converted(self) -> Tokenizer:
                 continuing_subword_prefix="",
                 end_of_word_suffix="</w>",
                 fuse_unk=False,
+                unk_token=str(unk_token),
             )
         )
 
-        tokenizer.pre_tokenizer = pre_tokenizers.ByteLevel(add_prefix_space=self.original_tokenizer.add_prefix_space)
+        tokenizer.normalizer = normalizers.Sequence(
+            [normalizers.NFC(), normalizers.Replace(Regex(r"\s+"), " "), normalizers.Lowercase()]
+        )
```

Comment:
Comment 2

Reply:
I think that for the normalization we have to add these 3 operations: 
- `NFC`, it is a unicode normalization that is used by `ftfy.fix_text ` (see corresponding code [here](https://github.com/rspeer/python-ftfy/blob/master/ftfy/__init__.py#L131-L140)) and that is quite common to add
- `Replace(Regex(r"\s+"), " ")`, as it is a normalization that is used by the `whitespace_clean` method defined in the slow tokenizer (see corresponding code [here](https://github.com/huggingface/transformers/blob/v4.15.0/src/transformers/models/clip/tokenization_clip.py#L95)) and that is quite common to add
- `Lowercase` as the text is always lowercase in the slow version (but see my question 6. in the introductory post)